src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentPlanCard/shared/PlanPricing/PlanPricing.test.jsx

```diff
@@ -20,116 +21,57 @@ describe('PlanPricing', () => {
     })
   })
 
-  describe('user is on a basic plan', () => {
```

Comment:
A lot of the tests in this file are obsolete now that we only care about the "type" of plan, annual vs. monthly and old vs. new plans don't matter in gazebo anymore